src/SFA.DAS.FindEpao.Api/Models/GetCourseEpaoStandardsListItem.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SFA.DAS.FindEpao.InnerApi.Responses;
```

Comment:
I think this file can be removed as well as it's not needed as part of the response

Reply:
removed